Python/Product/VSCode/.gitignore

```diff
@@ -0,0 +1,2 @@
+*/bin/
+*/obj/
```

Comment:
Rather than ignore these, can we force the output directories to be under the paths set up by the common targets? (or at least based on the same logic)

Reply:
It is when building from VSC C# extension (i.e. `dotnet build`). I'll see if I can make it drop output properly.